extensions/observability-services/runtime/src/main/resources/application.properties

```diff
@@ -0,0 +1,20 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+quarkus.micrometer.export.prometheus.path=/observe/metrics
+quarkus.smallrye-health.root-path=/observe/health
```

Comment:
Please can we add some basic tests for this into the `deployment` module ? See other tests under `extensions` for inspiration.

It can be simple as using REST Assured to hit the `/observe` endpoints and verifying there's a valid response.